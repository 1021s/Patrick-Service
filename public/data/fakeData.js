module.exports = {
  interiorDetails: {
    bedroomsAndBathrooms: {
      bedrooms: 5, bathrooms: 3, fullBathrooms: 3, halfBathrooms: 0,
    },
    basement: { basement: 'error est nemo' },
    flooring: { flooring: 'qui facilis omnis' },
    heating: { heating: 'et dolores asperiores, aut quis porro' },
    appliances: { appliances: 'maxime, impedit, et, blanditiis, optio' },
    otherInteriorFeatures: { livableArea: '2930 sqft' },
  },
  propertyDetails: {
    parking: { parkingFeatures: 'sint qui debitis', garageSpaces: 'doloremque minima aut' }, property: { stories: '1', exteriorFeatures: 'praesentium', viewDescription: 'soluta, perferendis, quod' }, lot: { lotSize: '7331 acres' }, otherPropertyInformation: { parcelNumber: '70555' },
  },
  constructionDetails: { typeAndStyle: { homeType: 'deleniti laboriosam dolore' }, materialInformation: { roof: 'harum atque dolorum' }, condition: { newConstruction: 'false', yearBuilt: '1983' } },
  utilitiesGreenEnergyDetails: { utility: { sewerInformation: 'laudantium temporibus qui', internetAndTv: 'sint beatae odio' }, greenEnergy: { sunscore: 'dolores voluptatem excepturi: 1' } },
  communityAndNeighborhoodDetails: { location: { region: 'Buckinghamshire' } },
  hoaAndFinancialDetails: { otherFinancialInformation: { taxAssessedValue: '$2655956', annualTaxAmount: '$36073' } },
  other: {
    sourceDetails: { mlsId: '2078077' },
    otherFacts: {
      busLineNearby: 'false', propertyType: 'recusandae', buildingInfo: 'saepe id incidunt', energySource: 'ut vel repudiandae', masterBedroomLocation: 'voluptates', waterSource: 'temporibus', foundation: 'nobis consequatur reiciendis', possesion: 'porro', parkingType: 'ullam autem illum', form17: 'aspernatur', appliancesThatStay: 'aut officiis adipisci', heatingAndCooling: 'blanditiis molestiae aut', entranceLevel: 'iusto', roof: 'ea', utilityRoomLocation: 'aut', features: 'velit facilis harum', lotDetails: 'aut aliquid cumque', floorCovering: 'vel voluptas sed', style: 'illo soluta aliquid', potentialTerms: 'reprehenderit ex id', kitchenLocation: 'velit', architecture: 'voluptas', lotTopography: 'illo', exterior: 'voluptate', siteFeatures: 'et odit repellat', offers: 'minus', commission: '4',
    },
  },
  _id: '5deb33a4a3a4fb338d482387',
  listingId: '067',
  type: 'qui amet provident',
  yearBuilt: 2008,
  heating: 'aut provident architecto',
  cooling: 'reprehenderit dolorem voluptates',
  parking: 'neque autem enim',
  lot: '3653 sqft',
  priceSqft: '$780',
  __v: 0,
};
