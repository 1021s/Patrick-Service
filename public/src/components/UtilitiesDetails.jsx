import React from 'react';

const UtilitiesDetails = ({ listing }) => (
  <div>

  </div>
);

export default UtilitiesDetails;
