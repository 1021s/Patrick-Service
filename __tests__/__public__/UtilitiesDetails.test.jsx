/* eslint-env jest */
/* eslint-env node */
import 'jsdom-global/register';
import React from 'react';
import '../../setupTests';
import { mount, shallow } from 'enzyme';
import fakeData from '../../public/data/fakeData';
import UtilitiesDetails from '../../public/src/components/UtilitiesDetails';

describe('UtilitiesDetails render suite', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<UtilitiesDetails debug />);
    expect(component).toMatchSnapshot();
  });

  it('should render correctly with props', () => {
    const component = mount(<UtilitiesDetails listing={fakeData.utilitiesGreenEnergyDetails} />);
    expect(component).toMatchSnapshot();
  });
});
