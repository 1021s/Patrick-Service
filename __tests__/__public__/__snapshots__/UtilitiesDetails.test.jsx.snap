// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UtilitiesDetails render suite should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;

exports[`UtilitiesDetails render suite should render correctly with props 1`] = `ReactWrapper {}`;
